script/bootstrap

```diff
@@ -8,3 +8,4 @@ cd "$(dirname "$0")/.."
 
 # Install node modules
 yarn install
+script/setup_translations
```

Comment:
I thought the goal was to use English only builds, unless the developer specifically wanted to fetch the translations?

Reply:
We can do that

Reply:
btw, last nights build failed with the new fetch translation script, do have time to check that? <https://github.com/home-assistant/frontend/actions/runs/3579496844>

Reply:
I'll take a look

Reply:
There's a problem with the backend translations.  I noticed this before but the error is new.  There are both `en-GB.json` and `en_GB.json` being downloaded, but only the underscore version is populated, which the gulp task ignores.  So the error is about reading an empty file.

Seems like it's either a core issue or a Lokalise issue with the backend project?

Reply:
I'll check Lokalise

Reply:
Fixed for `en-GB`, but now getting same problem with `ml.json` 😟

Reply:
That's weird, they seem to be the same in frontend en core...

Reply:
Ah, probably because nothing is translated in core

Reply:
So for some reason the `existsSync` check is failing when the file is empty even though it does exist.  That method is deprecated in node.js anyway, so we should fix by wrapping the write in a try/catch.

Reply:
no, the files dont exist in core, they do in frontend

Reply:
Oh... that makes more sense. 💡 

The fix is the same though.  It won't write because exists didn't fully close it yet (which is why it was deprecated).  I'll push a fix shortly but I cannot fully test since I can't download from the backend project.